Announce for OCaml 5.1.0~alpha1 (#1103)
ocaml/v3.ocaml.org: data/news/platform/ocaml-5.1.alpha1.md

---
title: OCaml 5.1.0 - First Alpha
description: First Alpha Release of OCaml 5.1.0
date: "2023-04-15"
tags: [ocaml, platform]
---

Four months after the release of OCaml 5.0.0, the set of new features for the

> future version 5.1.0 of OCaml has been frozen. we are thus happy to announce the

Reviewer: ```suggestion
future version 5.1.0 of OCaml has been frozen. We are thus happy to announce the
```

> first alpha release for OCaml 5.1.0 .

Reviewer: ```suggestion
first alpha release for OCaml 5.1.0.
```

This alpha version is here to help fellow hackers join us early in our bug
hunting and opam ecosystem fixing fun (see below for the installation instructions).

> The progresses on stabilizing the ecosystem are tracked on the [Opam Readiness for 5.1.0 meta-issue](https://github.com/ocaml/opam-repository/issues/23669).

Reviewer: ```suggestion
The progresses on stabilising the ecosystem are tracked on the [opam readiness for 5.1.0 meta-issue](https://github.com/ocaml/opam-repository/issues/23669) repo.
```

Author: I am not sure what you meant with the "repo" addition here?

> If you find any bugs, please report them on  [OCaml's issue tracker](https://github.com/ocaml/ocaml/issues).

Reviewer: ```suggestion
If you find any bugs, please report them on [OCaml's issue tracker](https://github.com/ocaml/ocaml/issues).
```

Note that this early alpha version is missing two important fixes for the garbage
collector and Windows support. Those fixes will be available before the beta.

> The full release is expected to happen in July.

Reviewer: ```suggestion
The full release is expected in July.
```

> If you are interested by the ongoing list of new features and bug fixes, the

Reviewer: ```suggestion
If you are interested in updates for the ongoing list of new features and bug fixes, the
```

Author: The changelog tracks the full list of changes, rather than just "updates". 
Maybe

> If you are interested in the ongoing list of new features and bug fixes

?

Reviewer: Perfect!
